interface/patient_file/encounter/find_code_history.php

```diff
@@ -79,10 +79,6 @@ function get_history_codes($pid)
     <meta charset="utf-8" />
     <?php Header::setupHeader(['opener']); ?>
     <style>
-        body {
-            height: 100%;
-            width: 100%;
-        }
         .tips {
```

Comment:
Try to replace this `tips` class with the BS4 `d-none` class in both the html and scripts.

Reply:
Be careful replacing it though cause there's an id called tips as well so you don't wanna replace that.